python/cudf/cudf/_libxx/io/io_types.pxd

```diff
@@ -22,14 +22,14 @@ cdef extern from "cudf/io/types.h" \
         QUOTE_NONE,
 
     ctypedef enum compression_type:
-        NONE,
-        AUTO,
-        SNAPPY,
-        GZIP,
-        BZIP2,
-        BROTLI,
-        ZIP,
-        XZ,
+        none "cudf::experimental::io::compression_type::NONE"
+        auto "cudf::experimental::io::compression_type::AUTO"
+        snappy "cudf::experimental::io::compression_type::SNAPPY"
+        gzip "cudf::experimental::io::compression_type::GZIP"
+        bzip2 "cudf::experimental::io::compression_type::BZIP2"
+        brotli "cudf::experimental::io::compression_type::BROTLI"
+        zip "cudf::experimental::io::compression_type::ZIP"
+        xz "cudf::experimental::io::compression_type::XZ"
 
     ctypedef enum io_type:
         FILEPATH,
```

Comment:
You need to give these the full C++ path afterwards, otherwise it doesn't use the C++ enum class IIRC

Reply:
Will make that change. I'm also going to change this filename to just "types.pxd" to match the new C++ filename as using "io_types.pxd" could confuse people into thinking this might be using the legacy C++ namespace